library/std/src/sys/unix/ext/fs.rs

```diff
@@ -358,6 +385,9 @@ impl OpenOptionsExt for OpenOptions {
         self.as_inner_mut().custom_flags(flags);
         self
     }
```

Comment:
```suggestion
    }

```
Add extra space here.